src/pages/index.js

```diff
@@ -134,5 +239,59 @@ profilePopup.setEventListeners();
 const cardPopup = new PopupWithForm(addModalClassStg, handleAddCardFormSubmit);
 cardPopup.setEventListeners();
 
+/*---------------------------------------------------*/
+/*          PopupWithImage Constructor               */
+/*---------------------------------------------------*/
+
 const popupImg = new PopupWithImage(previewModalClassStg);
 popupImg.setEventListeners();
+
+/*---------------------------------------------------*/
+/*         PopupWithCofirm Constructor               */
+/*---------------------------------------------------*/
+
+const trashConfirmPopup = new PopupWithConfirm(
+  trashModalClassStg,
+  handleConfirmModal
+);
+trashConfirmPopup.setEventListeners();
+
+/*---------------------------------------------------*/
+/*                      Api                          */
+/*---------------------------------------------------*/
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: authorizationCode,
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getProfile()
+  .then((data) => {
+    userInfo.setUserInfo(data.name, data.about);
+    avatarPic.src = data.avatar;
```

Comment:
`    avatarPic.src = data.avatar;
`

Class `UserInfo` should set the entire user data (including the **avatar**)